Source/CombatExtended/CombatExtended/Projectiles/ProjectileCE.cs

```diff
@@ -27,6 +27,11 @@ public abstract class ProjectileCE : ThingWithComps
         /// </summary>
         private const int collisionCheckSize = 5;
 
+        /// <summary>
+        /// Additional suppression multiplier for suppression caused by an explosion.
+        /// </summary>
+        private const float explosionSuppressionFactor = 3f;
```

Comment:
I'm unsure of the balance implications this factor has.

Reply:
Lower it to 2f if you think it's too high, gotta be aware of not nerfing it to near the previous state.